JobScheduler/appsettings.json

```diff
@@ -0,0 +1,20 @@
+{
+    "Logging": {
+        "LogLevel": {
+            "Default": "Information",
+            "Microsoft": "Warning",
+            "Microsoft.Hosting.Lifetime": "Information"
+        }
+    },
+    "App": {
+        "API": {
+            "ApiUrl": "https://localhost:5001/"
+        },
+        "IdentityServerConfig": {
+            "IdentityUrl": "https://localhost:5005/",
+            "ClientId": "dex-jobscheduler",
+            "ClientSecret": "dex-jobscheduler"
```

Comment:
I reckon we want to change this for production to an evironment variable that simultaneously gets used in the Identity Server? This way they always get matched and stay secret.